src/app/components/terms-and-conditions/terms-and-conditions.component.spec.ts

```diff
@@ -7,6 +7,8 @@ import { Store } from '@ngrx/store';
 import { of } from 'rxjs';
 import { TermsConditionsService } from '../../../shared/services/termsConditions.service';
 import { TermsAndConditionsComponent } from './terms-and-conditions.component';
+import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
+import { Router } from '@angular/router';
```

Comment:
incorrect import ordering

Reply:
Thank you @connorpgpmcelroy I updated the code now.